edge-apps/weather/index.html

```diff
@@ -41,31 +41,48 @@
                 </div>
                 <div class="weather-item dummy-node">
                   <span class="item-temp-degree">
-                    <span class="item-temp"></span><sup>°</sup>
+                    <span class="item-temp"></span>&deg;
                   </span>
                   <img class="item-icon" alt="clock icon">
                   <span class="item-time"></span>
                 </div>
             </section>
         </div>
-        <!--removeIf(sentry)-->
-        <script src="https://js.sentry-cdn.com/SENTRY_ID.min.js" crossorigin="anonymous"></script>
-        <!--endRemoveIf(sentry)-->
 
-        <!--removeIf(googleAnalytics)-->
-        <script async src="https://www.googletagmanager.com/gtag/js?id=GA_API_KEY"></script>
+        <script src="screenly.js?version=1"></script>
+
         <script>
-            window.dataLayer = window.dataLayer || [];
-            function gtag(){dataLayer.push(arguments);}
-            gtag('js', new Date());
+            const disableAnalytics = screenly.settings.disable_analytics == "true";
+
+            if (disableAnalytics != "true") {
+                const sentryId = screenly.settings.sentry_id;
+                const gaApiKey = screenly.settings.ga_api_key;
 
-            gtag('config', 'GA_API_KEY');
+                const sentryScript = document.createElement('script');
+                sentryScript.src = `https://js.sentry-cdn.com/${sentryId}.min.js`;
+                sentryScript.crossOrigin = 'anonymous';
+                document.head.appendChild(sentryScript);
+
+                const gtagScript = document.createElement('script');
+                gtagScript.src = `https://www.googletagmanager.com/gtag/js?id=${gaApiKey}`;
+                gtagScript.async = true;
+                document.head.appendChild(gtagScript);
+
+                window.dataLayer = window.dataLayer || [];
+                function gtag(){dataLayer.push(arguments);}
+
+                gtag('js', new Date());
+                gtag('config', gaApiKey);
+            }
         </script>
-        <!--endRemoveIf(googleAnalytics)-->
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.4/moment-with-locales.min.js" integrity="sha512-42PE0rd+wZ2hNXftlM78BSehIGzezNeQuzihiBCvUEB3CVxHvsShF86wBWwQORNxNINlBPuq7rG4WWhNiTVHFg==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+
+        <script
+            src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.4/moment-with-locales.min.js"
```

Comment:
Let's ship moment as part of the edge app.

Reply:
All done (locally). Expect changes on succeeding commit/s.